projects/vdk-core/src/vdk/internal/builtin_plugins/termination_message/writer.py

```diff
@@ -25,13 +30,25 @@ def vdk_configure(self, config_builder: ConfigurationBuilder):
 
     @hookimpl
     def vdk_exit(self, context: CoreContext, exit_code: int):
-        self._write_termination_message(
+        self.write_termination_message(
             errors.get_blamee_overall(),  # TODO: get this from context
             errors.get_blamee_overall_user_error(),  # TODO: get this from context
             context.configuration,
         )
 
-    def _write_termination_message(self, error_overall, user_error, configuration):
+    # Returns the version of the package named dist_name
+    @staticmethod
+    def version(
+        dist_name="vdk-core",
+    ):  # Change default value if project is renamed and does not equal the setuptools metadata.name
+        try:
+            return get_distribution(dist_name).version
+        except DistributionNotFound:  # pragma: no cover
+            return "unknown"
```

Comment:
Why not just use the `_version` function from the version module? Seems preferable to duplicating a chunk of code. If you agree with my comment, you might also want to rename that function to just `version`, as it is no longer a protected function.